### PR TITLE
Adding (uat_)contract.json + tests and Readme

### PR DESCRIPTION
**Pull request by hanssv**
~Still not with the final data for the Mainnet contract (nor the Readme obviously...).~

### CODE REVIEW

**Comment by sennui** [comment]
Can we have simlink to Readme in some visible place?

File: data/aecore/.lima/Readme.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+We also need the root-hash of the Merkle tree (the tree contains all the
+Ethereum accounts and their respective balances).
+
+```
+...
+```
+
+Finally we need to compile the contract (available
+HERE)[https://github.com/LimeChain/aepp-token-migration-smart-contract/blob/9909f608e2ddfbe1f9c0e9820f45bcea170a955d/contracts/TokenMigration.aes],
```
Comment (sennui) [inline data/aecore/.lima/Readme.md]: I think the contract shoudl live in Aeternity github namespace

**Comment by hanssv** [comment]
Yes, should be linked in the release notes I think...

File: data/aecore/.lima/Readme.md (hunk 1 above)
Comment (hanssv) [reply data/aecore/.lima/Readme.md]: It is identical to `test/contracts/token_migration.aes` 🤔

Comment (hanssv) [reply data/aecore/.lima/Readme.md]: Maybe point there instead?!

Comment (hanssv) [reply data/aecore/.lima/Readme.md]: Done.

**Comment by tolbrino** [comment]
@hanssv So is this the final version? The description is confusing.

**Comment by nikita-fuchs** [comment]
@ThomasArts @hanssv I guess all this is somehow affected by https://devchat.aeternity.com/group/token-migration-non-transferable?msg=W7Fu6DqtHtxsFyAX4 ?

**Pull request merged**